src/screens/DiscoveryScreenV2/fragment/DomainFragment.js

```diff
@@ -262,20 +282,38 @@ const DomainFragment = ({
     });
     const data = isFirstTimeOpen
       ? [
-          ...followingDomains.map((item, index) => ({
-            ...item,
-            user_id_follower: item.user_id_follower
-          })),
           {separator: true},
-          ...unfollowingDomains.map((item, index) => ({
+          ...unfollowingDomains.map((item) => ({
             ...item,
             user_id_follower: item.user_id_follower
           }))
         ]
-      : [...newMapFollowedDomain, {separator: true}, ...newMapUnfollowedDomain];
+      : [{separator: true}, ...newMapUnfollowedDomain];
+
+    const firstData = isFirstTimeOpen
+      ? followingDomains.map((item) => ({
+          ...item,
+          user_id_follower: item.user_id_follower
+        }))
+      : newMapFollowedDomain;
 
     return (
       <FlatList
+        ListHeaderComponent={() => (
+          <>
+            <RecentSearch
+              shown={!withoutRecent || isFirstTimeOpen}
+              setSearchText={setSearchText}
+              setIsFirstTimeOpen={setIsFirstTimeOpen}
+            />
+            <AccordionView
+              data={firstData}
+              renderItem={renderItem}
+              activeSections={activeSections}
+              setActiveSections={setActiveSections}
+            />
+          </>
+        )}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The FlatList component now includes a ListHeaderComponent prop that renders a RecentSearch and AccordionView component. This is a good way to add static content at the top of your list.
```javascript
<FlatList
  ListHeaderComponent={() => (
    <>
      <RecentSearch
        shown={!withoutRecent || isFirstTimeOpen}
        setSearchText={setSearchText}
        setIsFirstTimeOpen={setIsFirstTimeOpen}
      />
      <AccordionView
        data={firstData}
        renderItem={renderItem}
        activeSections={activeSections}
        setActiveSections={setActiveSections}
      />
    </>
  )}
  // ...
/>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->